docs/content/en/guides/performance-management/performance-management/index.md

```
@@ -164,13 +164,13 @@ jobs:
 
       - name: Run Performance Test
-        uses: meshery-extensions/meshery-performance-action@master
+        uses: layer5io/meshery-performance-action@master
         with:
           provider_token: ${{ secrets.PROVIDER_TOKEN }}
           platform: docker
           profile_name: soak-test
 ```
 
-More configuration details of the action can be found [here](https://github.com/meshery-extensions/meshery-performance-action/blob/master/action.yml).
+More configuration details of the action can be found [here](https://github.com/layer5io/meshery-performance-action/blob/master/action.yml).
```

markdownlint-cli2 (0.23.2)

[warning] Link text should be descriptive

(MD059, descriptive-link-text)